README.md

```diff
@@ -49,21 +49,24 @@ Setting up PSQL:
    DROP USER task_admin;
    ```
 
-Setting up project:
-1. Clone repo
-2. Setup Python Interpreter (PyCharm will complain that project does not have one set up, proceed to project settings and setup your previously installed Python there as interpreter)
-3. Open terminal, make sure you're in virtual enviroment (.venv) and install required packages:
+Setting up project (using PyCharm):
+1. Open PyCharm
+2. Create project from version control
+3. Setup Python Interpreter and virtual environment
```

Comment:
Not everyone uses PyCharm. It would be better to create a virtual environment using the appropriate commands. You can use a tool like virtualenv for this purpose, for example.